File: Exercicios-Beecrowd/Kotlin/1036.kt
import java.util.*
import kotlin.math.pow

fun main(args: Array<String>) {


    val lista = readLine().toString().split(" ")
    val a=lista[0].toDouble()
    val b=lista[1].toDouble()
    val c=lista[2].toDouble()
    if (a<=0){
        println("Impossivel calcular")
    }
    else {

        val delta = b.pow(2) - 4 * (a * c)
        if (delta<0){
            println("Impossivel calcular")

        }
        else {
            val r1 = ((b * -1) + delta.pow(0.5)) / (2 * a)
            val r2 = ((b * -1) - delta.pow(0.5)) / (2 * a)
            println("R1 = ${"%.5f".format(r1)}")
            println("R2 = ${"%.5f".format(r2)}")
        }
    }

}

File: Exercicios-Beecrowd/Python/1036.py
a,b,c= input().split()
a= float(a)
b= float(b)
c= float(c)
delta=(b**2)-4*a*c
if delta<=0 or a<=0: print('Impossivel calcular')
else:
    R1= ((-b)+ (delta**0.5))/(2*a)
    R2= ((-b)- (delta**0.5))/(2*a)
    print(f'R1 = {R1:.5f}')
    print(f'R2 = {R2:.5f}')